addons/payment_stripe/controllers/main.py

```diff
@@ -155,6 +157,13 @@ def stripe_webhook(self):
             _logger.exception("unable to handle the notification data; skipping to acknowledge")
         return ''
 
+    @http.route(_merchantid_domain_association_url, type='http', auth='public', csrf=False)
+    def stripe_merchantid_domain_association(self):
+        """ Returns the contents of the domain association file."""
+        return file_open(
```

Comment:
does it need to be a file? can't we have simply a global variable?

Reply:
We could use a global variable but the string is 9115 characters long so I think that it will make the code less readable. Stripe gave it as a file probably for the exact same reason.